akka-http-core/src/main/java/akka/http/impl/util/Util.java

```diff
@@ -35,7 +36,8 @@
         return (Source<U, scala.Unit>)(Object) p;
     }
     public static scala.collection.immutable.Map<String, String> convertMapToScala(Map<String, String> map) {
-        return emptyMap.$plus$plus(scala.collection.JavaConverters.mapAsScalaMapConverter(map).asScala());
+        return MapHelpers.convertMapToScala(map);
+        //emptyMap.<String>concat(scala.collection.JavaConverters.mapAsScalaMapConverter(map).asScala());
```

Comment:
Is JavaConverters removed in 2.13 or why is this needed?

Reply:
JavaConverters is fine, but converting to an immutable map was tricky.

This used to be done by creating an immutable map and calling `++`, but on 2.13 `++` is no longer available and `concat` is there instead - which doesn't work on 2.12. After some fiddling without a satisifying result I figured it'd be nicer to just have separate implementations and moved them to `MapHelpers`.